src/components/App.js

```diff
@@ -1,8 +1,13 @@
 import React from "react"
 import { Route, Switch, BrowserRouter } from "react-router-dom"
```

Comment:
Switch is being imported but isn't being used here. No importing without using.

Reply:
update implemented in app.js